go/libkb/secret_store_file_test.go

```diff
@@ -218,3 +219,42 @@ func TestSecretStoreFileNoise(t *testing.T) {
 
 	require.False(t, bytes.Equal(lksec.Bytes(), corrupt.Bytes()))
 }
+
+func TestPrimeSecretStoreFile(t *testing.T) {
+	td, tdClean := testSSDir(t)
+	defer tdClean()
+
+	tc := SetupTest(t, "secret_store_file", 1)
+	defer tc.Cleanup()
+
+	mctx := NewMetaContextForTest(tc)
+	secretStore := NewSecretStoreFile(td)
+	err := PrimeSecretStore(mctx, secretStore)
+	require.NoError(t, err)
+}
+
+func TestPrimeSecretStoreFileFail(t *testing.T) {
```

Comment:
why not have this test instead as part of a signup/provisioning test to verify the user creation short circuits

Reply:
sounds like a good idea...

Reply:
working on this right now

Reply:
thanks!